app/api/v1/endpoints/upload.py

```
@@ -0,0 +1,40 @@
+from fastapi import APIRouter, HTTPException, File, UploadFile
+from pydantic import BaseModel
+from azure.storage.blob import BlobServiceClient
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)